internal/charmstore/search.go

```
@@ -366,3 +387,23 @@ func typeFilter(value string) elasticsearch.Filter {
 	}
 	return elasticsearch.NotFilter{bundleFilter}
 }
+
+// createSort creates an elasticsearch.Sort query parameter out of a Sort parameter.
+func createSort(s SortParam) elasticsearch.Sort {
+	var sort elasticsearch.Sort
+	switch s.Field {
+	case "name":
+		sort.Field = "Name"
+	case "owner":
+		sort.Field = "User"
+	case "series":
+		sort.Field = "Series"
+	}
```

We only fail silently internally. v4/search.go will reject anything that's not the three names it knows about so you get an error if you sak for anything else.